tools/motus/motus_merge.xml

```diff
@@ -0,0 +1,97 @@
+<tool id="mereg_mOTUs_tables" name="Merge" version="@TOOL_VERSION@+galaxy0" profile="21.05">
+    <description>Merge mOTUs aboundance or count tables</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="biotools"/>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+        #if $db.db_source == 'cached'
+            #set $DATABASE_DIR=$db.db_cached.fields.path
+        #else
+            #set $database_path = $db.database
```

Comment:
I think the only way how we can trick the 14 GB github limit would be to try something like:
Extract the DB in the test folder and do not extract again if it is there. Can we test that ? 

```
#import os
#set $database_dir_path = os.path.dirname(os.path.abspath($db.database))
#set $database_path = $db.database
# if not os.path.isdir($database_dir_path/db_mOTU)
            tar -xvf '$database_path' -C '$database_dir_path' &&
```